ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-telemetry-postcard.j2

```diff
@@ -0,0 +1,72 @@
+{%             if monitor_telemetry_postcard.ingress.collection.version is arista.avd.defined(1) %}
+   ingress collection gre source {{ monitor_telemetry_postcard.ingress.collection.source }} destination {{ monitor_telemetry_postcard.ingress.collection.destination }}
+{%             else %}
+   ingress collection gre source {{ monitor_telemetry_postcard.ingress.collection.source }} destination {{ monitor_telemetry_postcard.ingress.collection.destination }}
+{%             endif %}
+{%         endif %}
+{%     endif %}
+{%     if monitor_telemetry_postcard.ingress.sample.rate is arista.avd.defined %}
+   ingress sample rate {{ monitor_telemetry_postcard.ingress.sample.rate }}
+{%     endif %}
+{%     if monitor_telemetry_postcard.ingress.sample.tcp_udp_checksum is arista.avd.defined %}
+   ingress sample tcp-udp-checksum {{ monitor_telemetry_postcard.ingress.sample.tcp_udp_checksum }}
+{%     endif %}
+{%     if monitor_telemetry_postcard.profiles is arista.avd.defined %}
+{%         for profile in monitor_telemetry_postcard.profiles %}
+   profile {{ profile.name }}
+{%             if profile.ingress_sample_policy is arista.avd.defined %}
+      ingress sample policy {{ profile.ingress_sample_policy }}
+{%             endif %}
+{%         endfor %}
+{%     endif %}
+{%     if monitor_telemetry_postcard.sample_policies is arista.avd.defined %}
+{%         for policy in monitor_telemetry_postcard.sample_policies %}
+{%             if policy.name is arista.avd.defined %}
+   sample policy {{ policy.name }}
+{%             endif %}
+{%             if policy.match_rules is arista.avd.defined %}
+{%                 for rule in policy.match_rules %}
+{%                     if rule.type is arista.avd.defined and rule.rule_name is arista.avd.defined %}
+      match rule {{ rule.rule_name }} {{ rule.type }}
+{%                     endif %}
+{%                     if rule.destination_prefix is arista.avd.defined %}
+         destination prefix {{ rule.destination_prefix }}
+{%                     endif %}
+{%                     if rule.source_prefix is arista.avd.defined %}
+         source prefix {{ rule.source_prefix }}
+{%                     endif %}
+{%                     if rule.protocol is arista.avd.defined and rule.destination_port is arista.avd.defined %}
+{%                         for port in rule.destination_port | sort(reverse = True) %}
+         protocol {{ rule.protocol }} destination port {{ port }}
+{%                         endfor %}
+{%                     endif %}
+{%                     if rule.protocol is arista.avd.defined and rule.source_port is arista.avd.defined %}
+{%                         for port in rule.source_port | sort(reverse = True) %}
+         protocol {{ rule.protocol }} source port {{ port }}
+{%                         endfor %}
+{%                     endif %}
+{%                 endfor %}
+{%             else %}
+      match ipv4-all-default ipv4
+      !
+      match ipv6-all-default ipv6
```

Comment:
So these two lines are printed if no match rules are configured? Normally we avoid having defaults in eos_cli_config_gen, and even with this, the behavior is not correct, since one of them might be manually configured above, and then the other one is auto-generated by EOS. So I would prefer if we just remove this `else` block.

Reply:
Removed else block.